packages/fdc3-get-agent/test/support/TestServerContext.ts

```diff
@@ -29,7 +33,7 @@ export class TestServerContext implements ServerContext<ConnectionDetails> {
     this.cw = cw;
   }
 
-  async narrowIntents(_raiser: AppIdentifier, appIntents: AppIntent[], _context: Context): Promise<AppIntent[]> {
+  async narrowIntents(_raiser: AppIdentifier, appIntents: AppIntent[] /*, _context: Context*/): Promise<AppIntent[]> {
```

Comment:
Should the commented-out argument be deleted?

Reply:
Its specified in the interface being implemented, but not used here. I thought it best to leave it in but commented for now.